src/components/recruit/HeaderSection.tsx

```diff
@@ -0,0 +1,73 @@
+import Image from 'next/image';
+import { css } from '@emotion/react';
+
+import { colors, mediaQuery } from '~/styles/constants';
+
+import { BigArrowIcon } from '../home/BigArrowIcon';
+
+export default function HeaderSection() {
+  return (
+    <section css={headerCss}>
+      <div css={headImageWrapperCss}>
+        <div css={headingCss}>
+          <h1>서류 접수 마감까지</h1>
+          <em>
+            <h1>D-7</h1>
+          </em>
+        </div>
+        <Image fill src="/images/recruit/home.png" alt={'리쿠르트 홈 아이콘 이미지'} />
```

Comment:
아직 일부러 안하신거 같지만 나중에 옵션 넣어주시면 좋을 거 같아요!!

```suggestion
        <Image 
          fill 
          src="/images/recruit/home.webp" 
          alt={'리쿠르트 홈 아이콘 이미지'} 
          placeholder="blur"
          blurDataURL="/images/recruit/home.webp" 
          priority
          quality={100}
        />
```

더 빨리 로드되게...!

> 아 그리고 webp 확장자가 용량이 훨씬 적더라구요!!

Reply:
오케 확인했습니당!